### ISSUE TITLE
iOS - Simulator vs Device nuget exceptions

### ISSUE THREAD

**Issue by ryansmithsquaresix** [RadiusNetworks/flybuy-ios-xamarin#9]
The nuget 2.1.5 is only compatible with running on a device, if a developer tries to debug on simulator it causes build and compilation errors

The nuget 2.1.4 is only compatible with running on a simulator, if a developer tries to debug on device it causes build and compilation errors

Can we expect a new nuget version that will work on both iOS device and simulators?

**Comment by ryansmithsquaresix** [comment]
@ajhewitt

**Comment by ajhewitt** [comment]
Hi @ryansmithsquaresix We had issues getting the XCFrameworks to work for both `arm` and `x86_64` platforms in a single Nuget, so the current bindings were built for `arm`. I can take another look to see if we can combine both platforms in one Nuget. If not then I can publish a build for `x86_64`.

**Comment by ryansmithsquaresix** [comment]
Thanks, I see a new nuget update, are you still looking into this issue or is it resolved?

**Comment by ajhewitt** [comment]
I did some research and confirmed Xamarin does not officially support xcframeworks. Our SDK structure and build process can only produce an xcframework, so I had to go the route of having two Nuget packages.

The `sim` version can be used for development with the simulator and then the regular version for production. It's a bit of a pain, but the only way forward at the moment. I will check back on this in the future and remove the `sim` build if the xcframework is fully supported.
